src/core/events/dispatcher.ts

```diff
@@ -21,18 +22,33 @@ export default class Dispatcher {
     return this;
   }
 
-  bind_all(callback : Function) {
+  bind_global(callback : Function) {
     this.global_callbacks.push(callback);
     return this;
   }
 
-  unbind(eventName : string, callback : Function, context?: any) {
+  unbind(eventName? : string, callback? : Function, context?: any) {
```

Comment:
According to the existing documentation and the underlying function, being able to call `unbind("event-name")` or `unbind()` is an intended use, so I just thought the types of the parameters in the typescript should respect that. It doesn't make any changes in the js! Totally agreed on it being kinda dangerous though. Would be a much bigger change to change that behaviour.